content/en/monitors/types/process.md

```diff
@@ -43,10 +66,16 @@ After defining your search, a graph is displayed above the search inputs with an
 
 ### Set alert conditions
 
-* The count was `above`, `above or equal to`, `below`, or `below or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 1 minute and 48 hours.
+- The count was `above`, `above or equal to`, `below`, or `below or equal to`
```

Comment:
I would also specify that "count" here means the count of all matching processes that were alive during the time interval

Reply:
Good point, added!